server/src/test/java/org/elasticsearch/search/aggregations/bucket/adjacency/AdjacencyMatrixAggregatorTests.java

```diff
@@ -40,14 +39,7 @@ public void testTooManyFilters() throws Exception {
         assertThat(
             ex.getMessage(),
             equalTo(
-                "Number of filters is too large, must be less than or equal to: ["
-                    + maxFilters
-                    + "] but was ["
-                    + maxFiltersPlusOne
-                    + "]."
-                    + "This limit can be set by changing the ["
-                    + SearchModule.INDICES_MAX_CLAUSE_COUNT_SETTING.getKey()
-                    + "] setting."
+                "Number of filters is too large, must be less than or equal to: [" + maxFilters + "] but was [" + maxFiltersPlusOne + "]."
```

Comment:
Let's add guidance as to how to address this error? E.g. can we check if the user overrode the size of the search thread and suggest undoing the change? Or otherwise recommend scaling up or avdoiding running queries with such large numbers of clauses?